packages/card-service/src/openapi/specs/card-server.yaml

```diff
@@ -0,0 +1,120 @@
+openapi: 3.0.3
+info:
+  title: Card Service API
+  version: 1.0.0
+paths:
+  /payment:
+    post:
+      summary: Accept a payment request from POS
+      operationId: createPayment
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - requestId
+                - card
+                - merchantWalletAddress
+                - incomingPaymentUrl
+                - date
+                - signature
+                - terminalCert
```

Comment:
Terminal cert is out of scope for now - maybe just remove the validation from the Koa server for now, or just delete this field.